transformer_engine/pytorch/tensor/mxfp8_tensor.py

```diff
@@ -552,7 +552,7 @@ def __torch_dispatch__(cls, func, types, args, kwargs=None):
                 rowwise_scale_inv=rowwise_scale_inv,
                 columnwise_data=columnwise_data,
                 columnwise_scale_inv=columnwise_scale_inv,
-                quantizer=tensor._quantizer.copy(),
```

Comment:
We should think more carefully before committing to this change. It doesn't make sense for the input and output of `zeros_like` to be entangled.